### DISCUSSION TITLE
Llama2 loop on incomplete prompts

### DISCUSSION THREAD

**Discussion by loretoparisi** [ggml-org/llama.cpp#2954] [General]
I did not experience this issue on not GGUF Llama weights. At the prompt the model loops forever until it crashes.
Model weights are `LLaMA-2-7B-32K-Q3_K_L.gguf`. If I remember well this happened in the past with older Llama GGML weights.
```
> read this text and answer the questions:
AI: 

### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :



### :


```

**Comment by mirek190** [comment]
I noticed loops on wizard 70B q4k_m and airboro_creative 70B q4k_m lately ... not always but from time to time ( I instructed to write a story ).
I never experienced that earlier. with ggml.bin model with llama 1s .
Maybe context size of wrong or something ... I set ctx to 4096  for llama2 but is still something off ...
